hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+import os
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
Please don't keep repeating this pattern, I believe it to be an antipattern.

logging.getLogger() will get the "root" logger
logger.setLevel() will change the  level of the "root" logger.

Most applications want to control the overall logging behavior of the application by changing settings on the root logger. This will overwrite that.

If you do want use logger, do the following

```
logger = logging.getLogger(__name__)
# don't set level

# or
def get_logger():
   """This pattern prevents creates implicitly creating a root logger by creating the sub-logger named __name__"""
  return logging.getLogger(__name__)
```

Reply:
All of our lambda that do logging have this pattern... if I don't set the level somewhere it won't generate the specific logs. for the `def get_logger()` options, can this code be shared between lambdas?